earth_enterprise/SConstruct

```diff
@@ -26,6 +26,8 @@ EnsureSConsVersion(0, 96, 92)
 
 # process commandline arguments
 installdir = ARGUMENTS.get('installdir', '/tmp/fusion_os_install')
+with open('.buildinfo', 'w') as f:
```

Comment:
I think we should pass this on the command line, or use a Gradle settings file, rather than implementing our own.

Reply:
@tst-ppenev  This is an entirely valid point of view.  I would be happy to discuss this.

Reply:
The way it used to work before is that the `stageOpenGeeInstall` task would copy the install files under the build directory.

Passing a value in a file creates a number of possible problems.  E.g., you can get undefined behavior if you try to run two tasks that use the file concurrently.

You can just add another project property.  E.g., `stageInstallDirectory`, and pass it on the command line like the `buildOpenGee` one.

On another note, I'm not sure it's a good idea to have tasks for both compiling and packaging in the same Scons file.  It may be contributing to poor isolation.